llvm/test/CodeGen/RISCV/rvv/fixed-vectors-fp-buildvec.ll

```diff
@@ -1828,3 +1828,110 @@ define <8 x double> @buildvec_v8f64_zvl512(double %e0, double %e1, double %e2, d
   %v7 = insertelement <8 x double> %v6, double %e7, i64 7
   ret <8 x double> %v7
 }
+
+define <8 x double> @buildvec_slideup(<4 x double> %v, double %e0, double %e1, double %e2, double %e3, double %e4, double %e5, double %e6) vscale_range(4, 128) {
```

Comment:
Should we add a test where the extract is the last non-undef element, but not the last element?